src/components/PlaylistModal/CreatePlaylistModal.jsx

```diff
@@ -0,0 +1,23 @@
+import "./PlaylistModal.css";
+import { useState } from "react";
+import {useVideos} from "../../context/video-context";
```

Comment:
You can use barrel imports for video context

Reply:
Thank you for the suggestion, will read about it.